### DISCUSSION TITLE
How to transform the Seurat object into ArchR object?

### DISCUSSION THREAD

**Discussion by wanglu2014** [GreenleafLab/ArchR#1967] [Questions / Documentation]
Dear developers:
After your answer, I have found a new Seurat object contains all the related ATAC information. Is it possiable to transform the Seurat object into ArchR object?
Thank you.
![图片](https://github.com/GreenleafLab/ArchR/assets/12541282/e741a9fc-c7ff-4108-9ac3-5e3764e6c697)

**Comment by uqnsarke** [comment]
following

**Comment by uqnsarke** [comment]
following
